functions/create_draft/interaction_handler.ts

```diff
@@ -22,7 +22,7 @@ import DraftDatastore from "../../datastores/drafts.ts";
  */
 
 export const openDraftEditView: BlockActionHandler<
-  typeof CreateDraftFunction.definition
+  typeof CreateDraftFunctionDefinition.definition
```

Comment:
@hello-ashleyintech - Good point! I've have put in a second PR that renames `definition.ts` to `mod.ts`. Which improves the import readability, a related issue as in: 

instead of 

```ts
import { CreateDraftFunctionDefinition } from "functions/create_draft/definition.ts". 
```

we have: 

```ts
import { CreateDraftFunctionDefinition } from "functions/create_draft/mod.ts". 
```
However you're right that in this case it does lead to some redundancy down below... I've added a bit at the import to rename it. 

```ts
import { CreateDraftFunctionDefinition as CreateDraftFunction } from "functions/create_draft/mod.ts". 

export const saveDraftEditSubmission: ViewSubmissionHandler<
  typeof CreateDraftFunction.definition
> = async (
  { inputs, view, client },
) => {
 ....
};
```